src/Infrastructure.VS.UnitTests/Infrastructure.VS.UnitTests.csproj

```diff
@@ -8,6 +8,10 @@
     <AssemblyName>SonarLint.VisualStudio.Infrastructure.VS.UnitTests</AssemblyName>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Would `<RequireLanguageServices>true</RequireLanguageServices>`  solve the problem? https://github.com/SonarSource/sonarlint-visualstudio/blob/1fa7a8f9fa7497ee9f6ab09c482993773cac02da/src/SonarLint.VSSpecificAssemblies.props#L108

Reply:
It doesn't. I have a reference to IAnalyzerAssemblyLoader and to AnalyzerFileReference, which needs the reference to Microsoft.CodeAnalysis